src/LanguageServer/Protocol/ProtocolConstants.cs

```diff
@@ -17,5 +17,7 @@ internal class ProtocolConstants
         public const string TypeScriptLanguageContract = "TypeScriptLspLanguage";
 
         public const string RazorCohostContract = "RazorLanguageServer";
+
+        public const string RazorFeaturesContract = "RazorFeatures";
```

Comment:
Did this end up being used anywhere?

Reply:
It did not. I'm surprised unused const doesn't have some kind of analyzer...